docs/java-rest/high-level/apis/search.asciidoc

```diff
@@ -178,3 +189,51 @@ cases need to be case accordingly:
 --------------------------------------------------
 include-tagged::{doc-tests}/SearchDocumentationIT.java[search-hits-singleHit-source]
 --------------------------------------------------
+
+==== Accessing Aggregations
```

Comment:
I think it is fine how you have it. Maybe link from the section above to this one.